gen/calc.py

```diff
@@ -387,7 +387,7 @@ def calculate_config_yaml(user_arguments):
 def calculate_mesos_isolation(enable_gpu_isolation):
     isolators = ('cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,'
                  'docker/runtime,docker/volume,volume/sandbox_path,posix/rlimits,'
-                 'namespaces/pid,com_mesosphere_MetricsIsolatorModule')
+                 'namespaces/pid,linux/capabilities,com_mesosphere_MetricsIsolatorModule')
```

Comment:
It would be nice to have a dcos-integration-test where Marathon or mesos-execute take advantage of the new capabilities.